pkg/interactor/data.go

```diff
@@ -0,0 +1,59 @@
+package interactor
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/data"
+	. "github.com/CESARBR/knot-cloud-storage/pkg/entities"
+)
+
+type DataInteractor struct {
+	DataStore data.DataStore
+}
+
+func NewDataInteractor(dataStore data.DataStore) DataInteractor {
+	return DataInteractor{dataStore}
+}
+
+func (d *DataInteractor) GetAll(order int, skip int, take int, startDate time.Time, finishDate time.Time) ([]Data, error) {
+
+	selectOrder := "timestamp"
+	if order == -1 {
+		selectOrder = "-timestamp"
+	}
+
+	data, err := d.DataStore.Get(selectOrder, skip, take, startDate, finishDate)
+
+	return data, err
+}
+
+func (d *DataInteractor) GetByID(id string, order int, skip int, take int, startDate time.Time, finishDate time.Time) ([]Data, error) {
```

Comment:
There's no need to repeat types if they're the same :grimacing: 

```suggestion
func (d *DataInteractor) GetByID(id string, order, skip, take int, startDate, finishDate time.Time) ([]Data, error) {
```

Reply:
Done!